ansible_collections/arista/avd/roles/eos_cli_config_gen/docs/tables/router-service-insertion.md

```diff
@@ -0,0 +1,16 @@
+<!--
+  ~ Copyright (c) 2023 Arista Networks, Inc.
+  ~ Use of this source code is governed by the Apache License 2.0
+  ~ that can be found in the LICENSE file.
+  -->
+=== "Table"
```

Comment:
Remember to add this table to the input variables md file manually.

Reply:
[9bcca08](https://github.com/aristanetworks/ansible-avd/pull/3220/commits/9bcca081daa178f07fc19cc3e153c968a3a61907)

Added STUN as well which was missed in #3147